docs/modules/ROOT/pages/index.adoc

```diff
@@ -408,12 +446,33 @@ The validator SHOULD verify that no entry in `signer_payload.referenced_assertio
+The validator MUST maintain a list of valid `signer_payload.sig_type` values and corresponding code paths for the `signature` values that it is prepared to accept. Validators SHOULD be prepared to accept all signature types described in xref:_credentials_signatures_and_validation_methods[xrefstyle=full]. The `cawg.identity.sig_type.unknown` error code SHALL be used to report assertions that contain unrecognized `signer_payload.sig_type` values.
 
 The `signature` field of an *<<_identity_assertion,identity assertion>>* MUST contain a valid signature. The procedure for validating each signature type and corresponding status codes are described in xref:_credentials_signatures_and_validation_methods[xrefstyle=full].
 
 The `pad1` and `pad2` fields of an *<<_identity_assertion,identity assertion>>* MUST contain only zero-value (`0x00`) bytes. The `cawg.identity.pad.invalid` error code SHALL be used to report assertions that contain other values in these fields.
 
+==== Validating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim` value in `signer_payload`, it MUST be validated as follows:
+
+. Locate the _<<C2PA claim>>_ data structure that includes this *<<_identity_assertion,identity assertion>>.*
+. Replace the `hash` value for the specific *<<_identity_assertion,identity assertion>>* being validated with all zero-value (`0x00`) bytes.
+. Serialize the updated _<<C2PA claim>>_ data structure using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized _<<C2PA claim>>_ data structure.
+
+The resulting hash MUST exactly match the value present in `expected_claim`. The `cawg.identity.expected_claim.mismatch` error code SHALL be used to report assertions that contain other values.
+
+==== Calculating the `expected_claim_generator` value
+
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim_generator` value in `signer_payload`, it MUST be validated as follows:
+
+. Locate the signing certificate chain that is present in the _<<C2PA claim>>_ signature. It can be located as described in link:https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#x509_certificates[Section 14.6, “X.509 certificates,”] of the C2PA technical specification.
+. Identify the CBOR bytes specific to this signing certifcate chain. If necessary, any re-serialization should be performed using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized certificate chain.
```

Comment:
Should this be computing a hash using the algorithm specified by `expected_claim_generator.alg`?